entity-framework/core/querying/client-eval.md

```diff
@@ -40,6 +40,9 @@ In such cases, you can explicitly opt into client evaluation by calling methods
 
 [!code-csharp[Main](../../../samples/core/Querying/ClientEvaluation/Program.cs#ExplicitClientEvaluation)]
 
+> [!TIP]
+> If you are using `AsAsyncEnumerable` and want to compose the query further on client side then you can use [System.Interactive.Async](https://www.nuget.org/packages/System.Interactive.Async/) library which defines operators for async enumerables.
```

Comment:
Add a link to https://docs.microsoft.com/en-us/ef/core/miscellaneous/async#client-side-async-linq-operators for more information